package/pyproject.toml

```diff
@@ -5,8 +5,7 @@ requires = [
     "packaging",
     # lowest NumPy we can use for a given Python,
     # except for more exotic platforms (linux/Mac ARM flavors)
-    "numpy==1.18; python_version=='3.7' and (platform_machine!='arm64' or platform_system!='Darwin') and platform_machine!='aarch64'",
-    "numpy==1.18; python_version=='3.8' and (platform_machine!='arm64' or platform_system!='Darwin') and platform_machine!='aarch64'",
+    "numpy==1.19; python_version=='3.8' and (platform_machine!='arm64' or platform_system!='Darwin') and platform_machine!='aarch64'",
```

Comment:
Any reason to bump the minimum NumPy version for `3.8`? Maybe not a big deal, though we're drifting even farther from https://github.com/scipy/oldest-supported-numpy/blob/main/setup.cfg#L53

Reply:
NEP29, if we only support 1.19 then the minimum version for build should be that version right?

Reply:
In support of @IAlibay :
```
$ nep29 numpy
| version |    date    |
|---------|------------|
|  1.22.0 | 2021-12-31 |
|  1.21.0 | 2021-06-22 |
|  1.20.0 | 2021-01-30 |
|  1.19.0 | 2020-06-20 |
```

Reply:
I guess I'm accustomed to less rigid adherence--I tend to follow the NEP29 Python schedule but maximize the NumPy support surface to prevent annoying stack rebuilds. Maybe less of a concern for MDA though, and if you can cite the NEP to avoid maintenance burden seems like a good idea.

Reply:
I general do favour your approach @tylerjereddy - however in this case I think bumping up to 1.19 is going to be useful to just be able to match conda-forge a little bit. If long term we'll just match whatever their numpy version defaults are for conda releases then it's best if we don't lag behind too much (since conda is the main way folks appear to install MDA).